src/test/java/org/mockito/configuration/MockitoConfiguration.java

```diff
@@ -0,0 +1,12 @@
+package org.mockito.configuration;
+
+/**
+ * This is to clean runners between tests.
+ */
+public class MockitoConfiguration extends DefaultMockitoConfiguration {
```

Comment:
@uKL havent changed the configuration. How do you configure this so that Mockito uses it?